sdk/identity/identity/test/public/node/azurePipelinesServiceConnection.spec.ts

```diff
@@ -2,12 +2,13 @@
 // Licensed under the MIT license.
 
 import { AzurePipelinesServiceConnectionCredential } from "../../../src";
-import { env, isLiveMode } from "@azure-tools/test-recorder";
+import { isLiveMode } from "@azure-tools/test-recorder";
 import { assert } from "@azure-tools/test-utils";
-
+import { setLogLevel } from "@azure/logger";
+setLogLevel("verbose");
 describe("AzurePipelinesServiceConnectionCredential", function () {
   const scope = "https://vault.azure.net/.default";
-  const tenantId = env.IDENTITY_SP_TENANT_ID || env.AZURE_TENANT_ID!;
+  const tenantId = "72f988bf-86f1-41af-91ab-2d7cd011db47";
```

Comment:
when we record these tests, you'd want any `env.` calls to be made after the recorder is initialized (so like in the test body or a `beforeEach` block after creating the recorder. Just FYI. I could be wrong but I remember this being a problem for me

Reply:
True. Though with this particular test, it's difficult to have recordings, since I cannot run locally :(